### PR TITLE
How do we validate a predictive model?

### PR DESCRIPTION
**Pull request by gottacatchenall**

### CODE REVIEW

**Comment by tpoisot** [comment]
I think this is still one question: what are the different approaches to validation and what do they tell us?

**Comment by gottacatchenall** [comment]
> I think this is still one question: what are the different approaches to validation and what do they tell us?

agreed that would make a better wording for the question, those questions i added are what i'm hoping to address in the answer---we'll see how it goes

**Comment by gottacatchenall** [comment]
@tpoisot @TanyaS08 this almost ready for review, but i'm looking for citation recommendations and any other comments before i attempt to merge

**Comment by tpoisot** [comment]
I suggest we merge after a review, and then when we integrate we can look at references. I like the fact that the current set of refs is technical and more hands-on discussions about model validation.

**Review by TanyaS08 — COMMENTED** [review]
Firstly, this is definitely on the better side of the scale of explaining model comparison/validation. Short and sweet but still informative. I'm assuming the plan is to expand on out of sample prediction? Also (out of curiosity) in one of the previous interactions hind casting was name dropped - is that a ship that has sailed/conversation for another day?

**Review by TanyaS08 — APPROVED** [review]
Happy to integrate as is if the rest of the review squad is

File: manuscript.md
Comment (tpoisot) [inline manuscript.md:35]: Can we turn this into full sentences?

File: manuscript.md
Comment (tpoisot) [inline manuscript.md:42]: Can we develop this a little bit more?

**Review by tpoisot — CHANGES_REQUESTED** [review]

**Comment by TanyaS08** [comment]
@gottacatchenall is this one ready for review - last commit says still in progress os don't want to flag for review juuuuust in case!

**Comment by gottacatchenall** [comment]
> @gottacatchenall is this one ready for review - last commit says still in progress os don't want to flag for review juuuuust in case!

there may be some minor changes to phrasing i want to make but that can be done at the integration stage---good to merge if you are

**Pull request merged**
